### DISCUSSION TITLE
[API Request] Support catalog number tags in AlbumID3 response

### DISCUSSION THREAD

**Discussion by atj** [opensubsonic/open-subsonic-api#103] [API requests]
### Type of change

API extension

### Proposal description

It would be nice to support the CATALOGNUMBER tag which is set by default in at least [MusicBrainz Picard](https://picard-docs.musicbrainz.org/en/variables/tags_basic.html#tags-provided-from-musicbrainz-data) and [MP3tag](https://docs.mp3tag.de/mapping/#catalognumber):

```
$ eyeD3 example.mp3 |grep -a CATALOGNUMBER -A1
UserTextFrame: [Description: CATALOGNUMBER]
ZENDNL192
$ metaflac --list example.flac |grep CATALOGNUMBER
    comment[11]: CATALOGNUMBER=WARPDD21R
```

The field can be multi-valued so it would be similar to `recordLabel` (e.g. an array of strings).

I may be in the minority but I find this information very useful, especially now that I am able to browse my collection via record label.

### Backward compatibility

- [X] No backward compatibility impact.

### API details

Add a new optional `catalogNumbers` field to the `AlbumID3` response:

```json
{
  "catalogNumbers": {
    "ZENDNL192"
  }
}
```

**Comment by bootybigboi** [comment]
Bumping this, I'd find this extremely valuable.
